tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -19,6 +19,23 @@ case "$ARCH" in
 	*) die "$ARCH not supported"
 esac
 
+if [ "${TEE_PLATFORM}" == "tdx" ] && [ "${ARCH}" == "x86_64" ]; then
+    source /etc/os-release
+
+    if [ "${OS_VERSION}" == "focal" ] || [ "${OS_VERSION}" == "20.04" ]; then
+        PACKAGES+=" apt gnupg ca-certificates"
+        PLATFORM_EXTRAS="
+    RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu ${OS_VERSION} main' \| tee /etc/apt/sources.list.d/intel-sgx.list; \
+        curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key \| apt-key add -; \
+        apt-get update; \
+        apt-get install -y libtdx-attest=1.15\* libtdx-attest-dev=1.15\* clang
```

Comment:
main branch TDX kernel upgraded to 6.2.x, dcap also need upgrade to 1.18

Reply:
fixed, thanks